db/deleteItem.go

```diff
@@ -0,0 +1,28 @@
+package db
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+)
+
+func (d *dynamoDB) DeleteItem(listID *string, itemID *string) error {
+	input := &dynamodb.DeleteItemInput{
+		Key: map[string]*dynamodb.AttributeValue{
+			"ListId": {S: listID},
+			"Id":     {S: itemID},
+		},
+		TableName: aws.String(d.conf.TableNames.Items),
```

Comment:
Will this be fine if `Items` is an empty string?

Reply:
Assume that dynamo will just fail? Would it be better to catch that early (before we make the request)?
Up to you